src/deadline/max_adaptor/MaxAdaptor/schemas/init_data.schema.json

```diff
@@ -20,15 +20,18 @@
                 ".tga",
                 ".tif",
                 ".dds",
-                ".cxr"
+                ".cxr",
+                ".vrimg"
             ]
         },
         "state_set": { "type": "string" },
         "renderer": {
             "enum": [
```

Comment:
Another option that may help better support more versions of VRay I believe is to do something like:
```
"renderer": {
    "oneOf": [
        {
            "type": "string",
            "pattern": "^V_Ray_[1-9](GPU)?"
        }
        {
            "enum": [
                "Default_Scanline_Renderer",
                "ART_Renderer",
                "Corona",
            ]
        }
    ]
}
```
No need to update for now unless you're interested in exploring that option, but it's something that might come up in the future. That would also allow setting the renderer to the full versioned renderer and not have to split at the "__". Then the `.split("__")` would only be needed when checking against the ALLOWED_RENDERERS list and otherwise it could be stored and displayed as the full string I believe.